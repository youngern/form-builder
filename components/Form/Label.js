import React from 'react';
import { Text, StyleSheet } from 'react-native';
import _ from 'lodash';
import Config from 'final-form-react-native/services/config';
const { Fonts } = Config;

const Label = ({ value }) => (
  <Text style={styles.label}>{_.upperCase(value)}</Text>
);

const styles = StyleSheet.create({
  label: {
    fontSize: 14,
    color: '#4d80e4',
    fontFamily: Fonts.Bold,
  },
});

export default Label;
