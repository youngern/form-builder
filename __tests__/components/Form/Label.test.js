import React from 'react';
import Label from 'final-form-react-native/src/components/Form/Label';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Label value={'test link'} />).toJSON();
  expect(tree).toMatchSnapshot();
});
