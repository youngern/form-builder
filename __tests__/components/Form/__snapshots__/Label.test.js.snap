// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#4d80e4",
      "fontFamily": "AppleSDGothicNeo-Bold",
      "fontSize": 14,
    }
  }
>
  test link
</Text>
`;
